front/pages/api/w/[wId]/dust_app_secrets/[name]/disable.ts

```diff
@@ -0,0 +1,73 @@
+import type { DustAppSecretType } from "@dust-tt/types";
```

Comment:
I think we want a DELETE endpoint on `front/pages/api/w/[wId]/dust_app_secrets/[name]` rather than a disable endpoint

Reply:
Yeah that's what I asked in the general comments of the PR - as we have a "don't delete anything" policy I went for disable, but I think in this use-case DELETE is the way :mandalorian:

Reply:
Even if we just disabled and not deleted for real, I'd still say the same thing :p better to have something more "REST-y"